_layouts/documentation.html

```diff
@@ -305,7 +305,7 @@
                             <ul class="nav nav-pills nav-stacked">
                                 <li
                                 {% if page.pill == 'configuration' %} class="nav--vertical__active" {% endif %}>
-                                <a href="/documentation/configuration/parameters">Overview</a></li>
+                                <a href="/documentation/configuration/parameters">Overview</a></li>                                                              
```

Comment:
A chunk of whitespace?

Reply:
chunk of dunno. i've selected it and hit delete but it shows up - im a bit lost with this.